spec/reek/spec/smell_matcher_spec.rb

```diff
@@ -55,8 +55,8 @@
       expect(matcher.matches_smell_type?(:UncommunicativeVariableName)).to be_truthy
     end
 
-    it 'matches on category symbol' do
-      expect(matcher.matches_smell_type?(:UncommunicativeName)).to be_truthy
+    it 'does not match on category symbol' do
```

Comment:
Same here, i'd rather remove it.
You're spec'ing for the negative of something, that doesn't exist anymore. Slightly confusing ;)